rskj-core/src/main/java/co/rsk/rpc/Web3InformationRetriever.java

```diff
@@ -44,8 +44,6 @@
  */
 public class Web3InformationRetriever {
 
-    private static final String EARLIEST = "earliest";
-    private static final String LATEST = "latest";
     private static final String PENDING = "pending";
```

Comment:
`PENDING` constant can also be replaced with the new enum you introduced?

Reply:
In this case this pending status was used with the transaction pool so I didn't want to mix it. is it ok to do it?

Reply:
I guess so. As I see it's treated as a block tag, so why not?

Reply:
I though maybe there were "transaction tags" as well